pipeline/vars/common.groovy

```diff
@@ -231,10 +242,11 @@ def sendEMail(def subjectPrefix, def test_results, def isStage=true) {
 
     emailext (
         mimeType: 'text/html',
-        subject: "${env.composeId} build is ${jobStatus} at QE ${subjectPrefix} stage.",
+        subject: "QE ${subjectPrefix} stage - ${jobStatus}",
```

Comment:
IMO, the subject line could be `Test report status of RH Ceph <major>.<minor> for {tier} is {status}`